build.gradle

```diff
@@ -4,38 +4,78 @@ plugins {
     id 'java'
     id 'com.gorylenko.gradle-git-properties' version '2.3.2'
     id 'com.adarshr.test-logger' version '3.2.0'
+    id 'io.miret.etienne.sass' version '1.4.2'
```

Comment:
Total curiosity here -- how did you decide to pick this package?

Reply:
I realize this is POC, too, so maybe you're still evaluating packages.

Reply:
I was looking for a package that connects [dark-sass](https://sass-lang.com/dart-sass) and puts a gradle wrapper over it.

There's not that many options.